Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.yml

```diff
@@ -0,0 +1,321 @@
+category: Network Security
+commonfields:
+  id: Microsoft 365 Defender
+  version: -1
+configuration:
+- additionalinfo: The API key to use to connect.
+  defaultvalue: 9093c354-630a-47f1-b087-6768eb9427e6
+  display: Application ID
+  name: app_id
+  required: true
+  type: 4
+- additionalinfo: |-
+    The United States: api-us.security.microsoft.com
+    Europe: api-eu.security.microsoft.com
+    The United Kingdom: api-uk.security.microsoft.co
+  defaultvalue: https://api.security.microsoft.com
+  display: Endpoint URI
+  hidden: false
+  name: base_url
+  required: true
+  type: 0
+- defaultvalue: 12 hours
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  hidden: false
+  name: first_fetch
+  required: false
+  type: 0
+- additionalinfo: The time limit in seconds for fetch incidents to run. Leave this
+    empty to cancel the timeout limit.
+  defaultvalue: '30'
+  display: Fetch incidents timeout
```

Comment:
please add this and newly added arguments such as `timeout` in the commands to the readme.

Reply:
Done